### PR TITLE
Adds VoiceChannels and Related Chats to Config

### LINKED ISSUE #1304: Add Missing Voice Channels To Config
The default config (and constants file) is missing the following channels:
- Code/Help 1 (voice)
- Code/Help 2 (voice)
- General/Offtopic (voice)
- code-help-voice-1 (text)
- code-help-voice-2 (text)
- staff-voice (text)
- admin-voice (text)

A lot of these channels either have the voice portion or the text portion, but not both. I've prepared a PR to address this.

### PR DESCRIPTION
**Pull request by HassanAbouelela**
Closes #1304

Adds the following channels to default config:
- Code/Help 1 (voice)
- Code/Help 2 (voice)
- General/Offtopic (voice)
- code-help-voice-1 (text)
- code-help-voice-2 (text)
- staff-voice (text)
- ~~admin-voice (text)~~
~~Note: admin-voice (text) is still missing an ID.~~

I tried sepearting out the two groups (text chat and voice chat), as it is confusing in the current form:
[L200-L201](https://github.com/python-discord/bot/blob/ed8160f/config-default.yml#L200-L201): voice
[L202-L204](https://github.com/python-discord/bot/blob/ed8160f/config-default.yml#L202-L204): text

### CODE REVIEW

**Comment by mbaruh** [comment]
It's necessary to merge this PR before #1297

File: config-default.yml [hunk 1]
```diff
@@ -196,10 +196,17 @@ guild:
...
         admin_announcements:    &ADMIN_ANNOUNCEMENTS    749736155569848370
 
-        # Voice
-        code_help_voice:                    755154969761677312
-        code_help_voice_2:                  766330079135268884
+        # Voice Chat
+        code_help_chat:                     755154969761677312
+        code_help_chat_2:                   766330079135268884
         voice_chat:                         412357430186344448
+        admins_voice_chat:                  000000000000000000 # FIXME
```
Comment (Den4200) [inline config-default.yml:203]: This channel doesn't actually exist.

File: config-default.yml [hunk 2]
```diff
@@ -196,10 +196,17 @@ guild:
...
+        staff_voice_chat:                   541638762007101470
+
+        # Voice Channels
+        code_help_voice:                    751592231726481530
```
Comment (Den4200) [inline config-default.yml:207]: I think `code_help_voice_1` may be a better name for this, following suite with the other code help voice channel.

**Review by Den4200 — CHANGES_REQUESTED** [review]

File: config-default.yml (hunk 2 above)
Comment (HassanAbouelela) [reply config-default.yml:207]: Agreed, I just left it in the original format to minimize changes, but I've updated it.

**Review by Den4200 — CHANGES_REQUESTED** [review]
Sorry for just realizing, the channels need to be alphabetized as well.

**Review by Den4200 — APPROVED** [review]
Looks good to me.

**Comment by ks129** [comment]
Not sure does this collide with https://github.com/python-discord/bot/pull/1016 .

**Review by ks129 — APPROVED** [review]
Looks good to me!

**Pull request merged**